pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -81,7 +81,9 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) *apps.Deployment {
 	if c.spec.Network.IsHost() {
 		podSpec.Spec.DNSPolicy = v1.DNSClusterFirstWithHostNet
 	} else if c.spec.Network.NetworkSpec.IsMultus() {
-		k8sutil.ApplyMultus(c.spec.Network.NetworkSpec, &podSpec.ObjectMeta)
+		if err := k8sutil.ApplyMultus(c.spec.Network.NetworkSpec, &podSpec.ObjectMeta); err != nil {
+			logger.Errorf("failed to apply multus spec to pod template metadata. %v", err)
```

Comment:
An err should be returned here. This means the `makeDeployment()` return signature will be `(*apps.Deployment, error)`, and the callers need to also return the error.